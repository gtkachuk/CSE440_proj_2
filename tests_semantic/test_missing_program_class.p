program missingProgramClass;


class someOtherClassButNotTheProgramClass

BEGIN
   
   VAR
      retval  : integer;
      counter : integer;

FUNCTION test;
BEGIN
      counter := 6
END

END
.
