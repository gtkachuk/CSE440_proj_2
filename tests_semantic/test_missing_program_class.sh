#!/bin/sh

. ../tests_lib.inc

test_diff_sem test_missing_program_class.p
