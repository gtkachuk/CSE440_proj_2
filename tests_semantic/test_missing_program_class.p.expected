ERROR:Missing program class
Errors detected. Exiting.
